service-seed-without-router/play-service/app/controllers/EchoController.java

```diff
@@ -17,10 +18,10 @@
      * @return CompletionStage of Result
      * @throws BaseException
      */
-    public CompletionStage<Result> echo(String id) throws BaseException {
+    public CompletionStage<Result> echo(String id,Http.Request req){
```

Comment:
AS we already have health controller , so why we need echo controller.

Reply:
@manzarul HealthController is not connected with Actor so that's why to demonstrate full functionality of the seeder-project we should have EchoController

Reply:
In that case correct health check only. 
if user call /health then it should check healthController and return respone.
if user call /service/health then it should go to health actor and return.